packages/reason-generate-types-from-graphql-schema/reason-generate-types-from-graphql-schema.0.7/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+maintainer: "greg <greg@hackages.io>"
+authors: "greg <greg@hackages.io>"
+build: [
+  ["ocamlbuild -r -use-ocamlfind  src/index.native"]
+]
+tags: ["reasonml" "graphql"]
+dev-repo: "https://github.com/Gregoirevda/reason-generate-types-from-graphql-schema.git"
+bug-reports: "https://github.com/Gregoirevda/reason-generate-types-from-graphql-schema/issues"
+homepage: "https://github.com/Gregoirevda/reason-generate-types-from-graphql-schema"
+install: ["./index.native"]
```

Comment:
do you want to install that file? In that case you need to either use a `cp` command (not really recommended) or generate a `.install` file (or let jbuilder do it for you).

Reply:
No, I don't want to install it. I thought it was required. Asking around let me understand it should list the binaries...